asmcomp/asmgen.ml

```diff
@@ -99,6 +99,12 @@ let rec regalloc ppf round fd =
     Reg.reinit(); Liveness.fundecl ppf newfd; regalloc ppf (round + 1) newfd
   end else newfd
 
+let cse fd =
+  if fd.Mach.fun_fast_compile then
```

Comment:
You should link at a relevant MPR to justify the choice of disabling specifically CSE.